testcases.py

```diff
@@ -996,6 +996,56 @@ def check(self) -> TestResult:
         return TestResult.FAILED
 
 
+class TestCaseNAT(TestCaseTransfer):
+    @staticmethod
+    def name():
+        return "nat"
+
+    @staticmethod
+    def abbreviation():
+        return "RN"
+
+    @staticmethod
+    def testname(p: Perspective):
+        return "transfer"
+
+    @staticmethod
+    def desc():
+        return (
+            "Transfer completes under frequent NAT port rebindings on the client side."
+        )
+
+    def get_paths(self):
+        self._files = [
+            self._generate_random_file(1 * MB),
+        ]
+        return self._files
+
+    @staticmethod
+    def scenario() -> str:
+        """ Scenario for the ns3 simulator """
+        return "nat --delay=15ms --bandwidth=10Mbps --queue=25 --first_rebind=1s --rebind_freq=3s"
```

Comment:
Strictly speaking, this doesn't guarantee a rebind during a transfer (and it risks a rebind during the handshake).
If we increase the file size to 10 MB or so, the transfer is guaranteed to take at least 8 seconds at this bandwidth, which would allow us to use safer values here.

Reply:
will do